components/Discussion/graphql/enhancers/withSubmitComment.js

```diff
@@ -31,10 +36,28 @@ export const withSubmitComment = compose(
         depth,
         focusId,
         discussionDisplayAuthor: displayAuthor,
-        discussionUserPreference: userPreference
+        discussionUserPreference: userPreference,
+        client,
+        includeParent
       },
       mutate
     }) => ({
+      previewComment: ({ content, discussionId, parentId, id }) => {
```

Comment:
Only set `previewComment` when `board` (probably already available in `ownProps` above) to prevent unnecessary request elsewhere